lib/hq/graphql/resource.rb

```diff
@@ -79,6 +79,10 @@ def sort_fields_enum
           @sort_fields_enum || ::HQ::GraphQL::Enum::SortBy
         end
 
+        def excluded_input_fields
+          @excluded_input_fields ||= ::HQ::GraphQL.excluded_inputs
```

Comment:
This probably shouldn't be memoized on initialization. It should probably just be something handled at run time, otherwise you have a pretty high chance you'll run into load order issues